src/components/pagination/index.md.njk

```diff
@@ -0,0 +1,107 @@
+---
+title: Pagination
```

Comment:
For simple language, we might want to call this component "Page numbers." Any thoughts all?

Reply:
"Page numbers" wouldn't cover the Previous and Next (with page titles) option...  "Page links" could work, but possibly a bit too vague.  I suspect "pagination" is widely known enough to be usable?

Reply:
I agree with Frankie. "Pagination" is a more widely used term and encompasses the different types of pagination this component supports.